ParisBNF/abb/BNFabb150.xml

```diff
@@ -28,6 +28,140 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <collection>d'Abbadie</collection>
                       <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b10087882s">BnF Éthiopien d'Abbadie 150</idno>
                     </msIdentifier>
+                  <msContents>
+                    <msItem xml:id="ms_i1">
+                        <locus from="2r" to="48rb"/>
+                        <title ref="LIT7560PrayerBE"/>
+                        <textLang mainLang="gez"/>
+                      <incipit xml:lang="gez"><hi rend="rubric">ይትባረክ፡ እ</hi>ግዚአብሔር፡<hi rend="rubric"> አምላ፡ እስራ</hi>ኤል፡ አምላክ፡ <hi rend="rubric">ለኵሉ፡ መንፈ</hi>ስ። ወለኵሉ፡  ዘሥጋ። በእንተ፡ ስብሐት። በእንተ፡ ሕፃናት፡ አምሩ፡ ስብሐትየ። ዝንቱ፡ መ<hi rend="rubric">ጽሐፍ፡ እግዚ</hi>እየ፡ እግዚእየ፡ በእ<hi rend="rubric">ንተ፡ ስምከ፡ መሐ</hi>ረኒ።</incipit>
+                      <explicit xml:lang="gez">አመ፡ ዕለተ፡ እጼውዓነ፡ ፍጡነ፡ ስምዓኒ። አፍቀርነ። እስመ፡ ሰምዓኒ፡ እግዚአብሔር፡ ቃለ፡ ጸሎትየ። ቆፍ፡ ጸራኅኩ፡ ኀቤከ፡ በኵሉ፡ ልብየ፡ እግዚኦ፡ ጸራኅኩ፡ ኀቤከ፡ ስምዓኒ፡ ወአድኅነኒ፡ ስምዓኒ፡ ቃልየ። እግዚኦ፡ ጸራኅኩ፡ ኀቤከ፡ ስምዓኒ፡ እግዚኦ፡ ጸሎትየ። ወአጽምዓኒ፡ ስእለትየ፡ በጽድቅከ፡ ፍጡነ፡ ስምዓኒ፡ እግዚኦ፡ ኃለፈት፡ ነፍስየ። ውእቱ፡ አግዚአብሔር፡ ውእቱ፡ ይስምዓኒ።</explicit>
+                        <colophon xml:id="coloph1">
+                        <q xml:lang="gez">በዝ፡ ጸሎት፡ ፈጸምኩ፡ በጸጋሁ፡ ወበረድኤቱ። ይትአኰት፡ ስመ፡ ለእግዚአብሔር። ነፍስየኒ፡ ወሥጋየኒ፡ በደኃሪት፡ ዕለት፡ መሐረኒ፡ ለገብርከ፡ <hi rend="rubric">መንክር፡</hi> አቡየኒ፡ <hi rend="rubric">መገዛሁ፡</hi> ወመምህረኒ፡ <hi rend="rubric">መዓዛሁ፡</hi> እምየኒ፡ <hi rend="rubric">ኒመህር፡</hi> ለዘጸሐፎ፡ ወለዘአጽሐፎ፡ ለዘአንበቦ፡ ወለዘተርገሞ፡ ወለዘሰምዓ፡ ቃላቲሁ፡ ኅቡረ፡ ይምሐረነ፡ እግዚአብሔር፡ ለዓለመ፡ ዓለም፡ አሜን፡ ወአሜን፡ ለይኩን፡ ለይኩን። ። ። ።
+                          </q>
+                      </colophon>
+                       <listBibl type="text">
+                          <bibl>
+                            <ptr target="bm:Aescoly1951Recueil"/>
+                            <citedRange unit="page">198–266</citedRange>
+                          </bibl>
+                        </listBibl>
+                    </msItem>
+                  </msContents>
+                  <physDesc>
+                    <objectDesc form="Codex">
+                      <supportDesc>
+                        <support>
+                          <material key="parchment"/>
+                        </support>
+                        <extent>
+                          <measure unit="leaf" quantity="48">48</measure>
+                          <dimensions type="outer">
+                            <height unit="mm">125</height>
+                            <width unit="mm">100</width>
+                          </dimensions>
+                          <measure unit="leaf" type="blank">1
+                            <locus target="#148v"></locus></measure>
```

Comment:
I think we usually put locus after measure in these cases, but I don't know if it makes a difference

Reply:
```
     <measure unit="leaf" type="blank">1</measure>
                            <locus target="#148v"/>
```